pegasus/sites.v3/code.org/views/elementary_school_curricula_video.haml

```diff
@@ -1,37 +1,40 @@
-- video_curricula_index = 5
+- show_game_design_curriculum = DCDO.get('curriculum-launch-2024', false)
 
-- video_curricula_tags = [hoc_s(:module_grade_k_5), hoc_s(:module_grade_3_5), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_k_12)]
+- video_curricula_index = 6
 
-- video_curricula_title = [hoc_s(:curriculum_name_csf), hoc_s(:curriculum_name_csc), hoc_s(:curriculum_name_artificial_intelligence), hoc_s(:curriculum_name_maker), hoc_s(:hour_of_code)]
+- video_curricula_tags = [hoc_s(:module_grade_k_5), hoc_s(:module_grade_3_5), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_k_12)]
 
-- video_curricula_desc = [hoc_s(:csf_page_top_desc), hoc_s(:csc_page_top_desc), hoc_s(:ai_hero_desc), hoc_s(:curriculum_desc_maker), hoc_s(:hoc_homepage_what_is_hoc_subhead)]
+- video_curricula_title = [hoc_s(:curriculum_name_csf), hoc_s(:curriculum_name_csc), hoc_s(:curriculum_name_artificial_intelligence), hoc_s(:curriculum_name_maker), hoc_s('curricula_pages.game_design.title'), hoc_s(:hour_of_code)]
+
+- video_curricula_desc = [hoc_s(:csf_page_top_desc), hoc_s(:csc_page_top_desc), hoc_s(:ai_hero_desc), hoc_s(:curriculum_desc_maker), hoc_s('curricula_pages.game_design.desc'), hoc_s(:hoc_homepage_what_is_hoc_subhead)]
 
 - video_id = ["cs_connections", "ai_intelligence", "courses_maker"]
-- video_link =["/csf", "/csc", "/ai", "/maker", "/hourofcode"]
-- video_link_title=[:call_to_action_explore_csf, :call_to_action_explore_csc, :ai_why_link_01, :call_to_action_explore_maker, :call_to_action_explore_hoc]
+- video_link =["/csf", "/csc", "/ai", "/maker", "/curriculum/games", "/hourofcode"]

```

Comment:
We changed this to `code.org/curriculum/game-design`. Moshe and Bethany signed off, but Dani hasn't yet since she's OOO so there's still a chance this won't change, but I made the page at `game-design.haml` ([comment ref](https://codedotorg.atlassian.net/browse/ACQ-1957?focusedCommentId=28983))
```suggestion
- video_link =["/csf", "/csc", "/ai", "/maker", "/curriculum/game-design", "/hourofcode"]

```